src/fiskaltrust.Middleware.Interface.Soap/fiskaltrust.Middleware.Interface.Soap.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net40;net461</TargetFrameworks>
```

Comment:
AFAIK the client should work with .netstandard2.0 too

Reply:
We'd need to add multiple dependencies then - all those System.ServiceModel.* stuff. Since the interface is currently also not supporting SOAP with netstandard, would it be okay for you to stick to net461?